cvmfs/swissknife_list_reflog.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_SWISSKNIFE_LIST_REFLOG_H_
+#define CVMFS_SWISSKNIFE_LIST_REFLOG_H_
+
+#include <set>
+#include <string>
+
+#include "catalog_traversal.h"
+#include "swissknife.h"
+
+using namespace std;  // NOLINT
+
+namespace swissknife {
+
+class CommandListReflog : public Command {
+ public:
+  ~CommandListReflog() { }
+  virtual string GetName() const { return "list_reflog"; }
```

Comment:
I used underscore since it is in the reconstruct_reflog command: https://github.com/cvmfs/cvmfs/blob/devel/cvmfs/swissknife_reflog.h#L21
Am I missing something?

Reply:
Oh, I see.  My bad, I confused it with the cvmfs-server UI.